lib/samuel/checks/has_comments.ex

```diff
@@ -33,16 +33,19 @@ defmodule Samuel.Checks.HasComments do
   end
 
   def requirements do
-    ~w(comments)a
+    ~w(commit_comments pr_comments)a
   end
 
+  defp all_comments(data) do
+    data.commit_comments ++ data.pr_comments
+  end
 
   # Returns the number of comments made by someone who is not the Pull Request
   # author or Samuel.
   defp other_user_comments(data) do
     users_that_dont_count = users_that_dont_count(data.event)
```

Comment:
It's one of my favourite things about ruby: if you see `x = foo` you don't know if `foo` is a method or a variable, and *the caller doesn't care* - it just wants the data.